pkg/apis/triggers/v1alpha1/cluster_interceptor_defaults.go

```diff
@@ -18,17 +18,7 @@ package v1alpha1
 
 import (
 	"context"
-
-	"github.com/tektoncd/triggers/pkg/apis/triggers/contexts"
 )
 
 // SetDefaults sets the defaults on the object.
-func (it *ClusterInterceptor) SetDefaults(ctx context.Context) {
```

Comment:
@dibyom Removed setting default port because earlier if no port specified the default port 80 use to set to svc.Port using setDefaults
But now default port will change based on caBundle and caBundle for clusterintercptor will be populated little late than webhook execution

So thats the reason removed defaulting from here and added as part of [cluster_interceptor_types.go](https://github.com/tektoncd/triggers/pull/1333/files?show-deleted-files=true&show-viewed-files=true&file-filters%5B%5D=#diff-dd761b819fad42dcd5732dbd8fc12671dd1caa5ad8b0d8bd1870a92861e9a303R130-R139) 

Scenario :

1. if port specified as part of clusterintercepor spec then use that port
2. If no port specified and cabundle is specified then use `8443` port as default
3. If no port and cabundle is specified then use `80` port as default